lib/cocoapods/executable.rb

```diff
@@ -53,7 +53,7 @@ def self.execute_command(executable, command, raise_on_failure)
 
       require 'open4'
 
-      full_command = "#{bin} #{command}"
+      full_command = "\"#{bin}\" #{command}"
```

Comment:
Unfortunately, shell escape doesn't do the trick, even more escaping is required :)

What works: `Open4.spawn('false\\\\\"\ e')`

Now we need to find an escape function that already works :palm_tree: